cloudformation/antmedia-aws-autoscale-template.yaml

```diff
@@ -367,216 +417,32 @@ Resources:
-      DependsOn:
-      - LaunchConfigOrigin
-      Properties:
-        VPCZoneIdentifier: !Ref Subnets
-        LaunchConfigurationName: !Ref LaunchConfigOrigin
-        MinSize: !Ref AntMediaOriginCapacity
-        MaxSize: !Ref AntMediaOriginCapacityMax
-        DesiredCapacity: !Ref AntMediaOriginCapacity
-        TargetGroupARNs:
-          - !Ref ALBTargetGroupOrigin
-        LoadBalancerNames: 
-           - !If [CreateRTMPResources, !Ref RTMPLoadBalancer, !Ref "AWS::NoValue"]
-        Tags:
-        - Key: Name
-          Value: Antmedia-Origin
-          PropagateAtLaunch: 'true'
-      CreationPolicy:
-        ResourceSignal:
-          Timeout: PT15M
-          Count: !Ref AntMediaOriginCapacity
-      UpdatePolicy:
-        AutoScalingRollingUpdate:
-          MinInstancesInService: '1'
-          MaxBatchSize: '1'
-          PauseTime: PT15M
-          WaitOnResourceSignals: 'true'
-        NotificationConfiguration:
-          TopicARN: !Ref NotificationTopic
-          NotificationTypes:
-            - 'autoscaling:EC2_INSTANCE_LAUNCH'
-            - 'autoscaling:EC2_INSTANCE_LAUNCH_ERROR'
-            - 'autoscaling:EC2_INSTANCE_TERMINATE'
-            - 'autoscaling:EC2_INSTANCE_TERMINATE_ERROR'
-  LaunchConfigOrigin:
-    Type: 'AWS::AutoScaling::LaunchConfiguration'
-    DependsOn:
-      - ApplicationLoadBalancer
-    Metadata:
-      Comment: Install a simple application
-      'AWS::CloudFormation::Init':
-        configSets:
-          setup:
-            - "configure_cfn"
-        configure_cfn:
-          files:
-            /etc/cfn/cfn-hup.conf:
-              content: !Sub |
-                [main]
-                stack=${AWS::StackId}
-                region=${AWS::Region}
-                verbose=true
-                interval=5
-              mode: "000400"
-              owner: root
-              group: root
-            /etc/cfn/hooks.d/cfn-auto-reloader.conf:
-              content: !Sub |
-                [cfn-auto-reloader-hook]
-                triggers=post.update
-                path=Resources.LaunchConfigOrigin.Metadata.AWS::CloudFormation::Init
-                action=/opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource LaunchConfigOrigin --configsets setup --region ${AWS::Region}
-              mode: "000400"
-              owner: root
-              group: root
-            /lib/systemd/system/cfn-hup.service:
-              content: !Sub |
-                [Unit]
-                Description=cfn-hup daemon
-                
-                [Service]
-                Type=simple
-                ExecStart=/opt/aws/bin/cfn-hup
-                Restart=always
-                
-                [Install]
-                WantedBy=multi-user.target
-              mode: "000400"
-              owner: root
-              group: root
+    Fn::Transform:
+      Name: AWS::Include
+      Parameters:
+        Location:
+          Fn::Sub: s3://ams-cloudformation/origin.yaml
```

Comment:
Where are this origin.yaml and other sub yaml located in our repo?

Reply:
They are located in our s3 public bucket.

Reply:
What if it's deleted accidently in s3 public bucket. I think they should be also in the github repo.

Reply:
If I remember correctly, we talked about putting them in S3. Please correct me if I'm wrong.

Reply:
Putting them in S3 is ok. 

I'm just asking what if they are deleted or needs to be changed? Isn't it a good idea to store in a repo so that there is a backup and we see the changes over previous ones.